eng/scripts/Armstrong-Validation.ps1

```diff
@@ -0,0 +1,200 @@
+[CmdletBinding()]
+param (
+  [Parameter(Position = 0)]
+  [string] $BaseCommitish = "HEAD^",
+  [Parameter(Position = 1)]
+  [string] $TargetCommitish = "HEAD"
+)
+Set-StrictMode -Version 3
+
+. $PSScriptRoot/ChangedFiles-Functions.ps1
+. $PSScriptRoot/Logging-Functions.ps1
+
+$script:psYamlInstalled = $false
+function Ensure-PowerShell-Yaml-Installed {
```

Comment:
@mikeharder if we are starting to use this suppression support more it seems like a good idea to move these to be common helpers.

Reply:
Already tracked: https://github.com/Azure/azure-rest-api-specs/issues/28069